### ISSUE TITLE
rundeck CLI scripts naming

### ISSUE THREAD

**Issue by jbguerraz** [rundeck/rundeck#2081]
# Bug report

**My Rundeck detail**
- Rundeck version: 2.6.9
- install type: rpm (rpm,deb,war,launcher)
- OS Name/version: fedora

**Expected Behavior**

https://github.com/rundeck/rundeck/tree/master/core/src/main/resources/com/dtolabs/rundeck/core/cli/templates

The scripts names are all prefixed with "rd-"

**Actual Behavior**

run and dispatch scripts are not prefixed with "rd-" which make them too general

**How to reproduce Behavior**

ls /usr/bin

**Comment by gschueler** [comment]
those cli tools are deprecated in favor of this https://github.com/rundeck/rundeck-cli

**Comment by jbguerraz** [comment]
So I won't provide them from alpine linux package then :) btw, why not to simply delete those tools then ?

**Comment by gschueler** [comment]
@jbguerraz once we think the new tool is ready we probably will remove the old tools

**Comment by dserodio** [comment]
Duplicate of #1896
